src/vma/dev/ring_simple.cpp

```diff
@@ -721,20 +721,24 @@ const char* priv_igmp_type_tostr(uint8_t igmptype)
 }
 
 // calling sockinfo callback with RFS bypass
-static inline bool check_rx_packet(sockinfo *si, mem_buf_desc_t* p_rx_wc_buf_desc)
+//
+static inline bool check_rx_packet(sockinfo *si, mem_buf_desc_t* p_rx_wc_buf_desc, void *fd_ready_array)
 {
 	// Dispatching: Notify new packet to the FIRST registered receiver ONLY
-	p_rx_wc_buf_desc->reset_ref_count();
+
 #ifdef RDTSC_MEASURE_RX_DISPATCH_PACKET
 	RDTSC_TAKE_START(g_rdtsc_instr_info_arr[RDTSC_FLOW_RX_DISPATCH_PACKET]);
 #endif //RDTSC_MEASURE_RX_DISPATCH_PACKET
 
+	p_rx_wc_buf_desc->reset_ref_count();
```

Comment:
ticket was added and assigned to you, potentially it can be moved but is it would be part of this ticket work.